src/main/java/no/fdk/dataservicecatalog/controller/ApplicationStatusHandler.java

```diff
@@ -0,0 +1,22 @@
+package no.fdk.dataservicecatalog.controller;
+
+import org.springframework.stereotype.Component;
+import org.springframework.web.reactive.function.server.ServerRequest;
+import org.springframework.web.reactive.function.server.ServerResponse;
+import reactor.core.publisher.Mono;
+
+import static org.springframework.web.reactive.function.server.ServerResponse.ok;
+
+@Component
+public class ApplicationStatusHandler {
+
+    public Mono<ServerResponse> ping(ServerRequest serverRequest) {
+        return ok().bodyValue("OK");
+    }
+
+    public Mono<ServerResponse> ready(ServerRequest serverRequest) {
+        return ok().bodyValue("OK");
```

Comment:
Her bør du sjekke litt mer en kun 200 respons. Sjekk om du når databasen for eks.

Reply:
Regner med at det blir et salig kaos i GCP om halve stacken går ned og begynner å skrike og restarter seg selv om mongodb er utilgjengelig en liten stund. Men @valosnah kan jo kommentere på hvordan han vil ha det.

Reply:
Sett den mot mongodb du. Det blir mest korrekt.

Reply:
04822c8